tests/Cooked/BalancingSpec.hs

```diff
@@ -34,15 +34,15 @@ banana = permanentValue "banana"
 initialDistributionBalancing :: InitialDistribution
 initialDistributionBalancing =
   InitialDistribution
-    [ paysScript (alwaysTrueValidator @MockContract) () (Script.ada 42),
-      paysPK alice (Script.ada 2 <> apple 3),
-      paysPK alice (Script.ada 25),
-      paysPK alice (Script.ada 40 <> orange 6),
-      paysPK alice (Script.ada 8),
-      paysPK alice (Script.ada 30),
-      paysPK alice (Script.lovelace 1280229 <> banana 3) `withDatum` (10 :: Integer),
-      paysPK alice (Script.ada 1 <> banana 7) `withReferenceScript` (alwaysTrueValidator @MockContract),
-      paysPK alice (Script.ada 105 <> banana 2) `withUnresolvedDatumHash` ()
+    [ alwaysTrueValidator @MockContract `receives` (Value (Script.ada 42) <&&> VisibleHashedDatum ()),
+      alice `receives` Value (Script.ada 2 <> apple 3),
```

Comment:
Same: do we now have to ways of doing the same thing (put differently, is `Value` an homomorphism from `(_, <>)` to `(_, <&&>)`) ? Not that it's a problem per se, I'm just wondering.

Reply:
In short: no it's not (see above).